app/models/katello/content_view_repository.rb

```diff
@@ -27,7 +31,7 @@ def content_view_composite
     end
 
     def ensure_repository_type
-      unless ALLOWED_REPOSITORY_TYPES.include?(repository.content_type)
+      unless allowed_repository_types.include?(repository.content_type)
```

Comment:
add a test for this

Reply:
See test/models/content_view_repository_test.rb